src/pudl/metadata/fields.py

```diff
@@ -3704,47 +3798,22 @@
     "storage_enclosure_code": {
         "type": "string",
         "description": "A code representing the enclosure type that best describes where the generator is located.",
-        "constraints": {
-            "enum": set(
-                CODE_METADATA["core_eia__codes_storage_enclosure_types"]["df"].code
-            )
-        },
     },
```

Comment:
why are all of these enums gone?

Reply:
These columns are naturally constrained by virtue of having a FK relationship with the coding tables. The `enum` is duplicative.  There are some other cases where we are using some *other* column that's not part of the FK relationship as a constraint, and those do need to be `enum`.